spec/factories/rounds.rb

```diff
@@ -27,5 +37,6 @@
     members_can_propose_buckets true
```

Comment:
but yeah, we never got around to actually implementing it properly. :stuck_out_tongue_closed_eyes: